File: WalkingMaps/WalkingMaps/wwwroot/app/core/domain/sight.ts
export class Sight {
    Id: number;
    Name: string;
    Description: string;
    NavLink: string;
    Address: string;
    CreatedDate: Date;
    FormattedAddress: string;
    Latitude: number;
    Longtitude: number;
    UserId: string;
    PhotoUri: string;

   

    constructor(id: number,
        name: string,
        description: string,
        navLink: string,
        address: string,
        createdDate: Date,
        formattedAddress: string,
        latitude: number,
        longtitude: number,
        userId: string,
        photoUri: string)
    {

        this.Id = id;
        this.Name = name;
        this.Description = description;
        this.NavLink = navLink;
        this.Address = address;
        this.CreatedDate = createdDate;
        this.FormattedAddress = formattedAddress;
        this.Latitude = latitude;
        this.Longtitude = longtitude;
        this.UserId = userId;
        this.PhotoUri = photoUri;
    }
}

File: WalkingMaps/WalkingMaps/Entities/Sight.cs
using System;
using System.Collections.Generic;

namespace WalkingMaps.Entities
{
    public class Sight : IEntityBase
    {
        public Sight()
        {
            SightWalks = new List<WalkSight>();
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string NavLink { get; set; }
        public string Address { get; set; }
        public DateTime CreatedDate { get; set; }
        public string FormattedAddress { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string UserId { get; set; }      
        public virtual ICollection<WalkSight> SightWalks { get; set; }
        public string PhotoUri { get; set; }
        public string Author { get; set; }
    }
}
